package.json

```
@@ -20,7 +20,7 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/atom/docs.git"
+    "url": "https://github.com/pulsar-edit/docs.git"
```

```suggestion
    "url": "https://github.com/pulsar-edit/flight-manual.atom.io.git"
```

 I know this isn't what was done originally, but makes a lot more sense to me imo.